### PR TITLE
5 Reporting - v4 Migration Cleanup - Closes #89

### LINKED ISSUE #89: 5 Reporting - v4 Migration Cleanup
- [ ] "wikilinks" during migration the OWASP Mediawiki links were tagged "wikilink" these needs to be converted to proper absolute URLs.
  - Unless the wikilink is to content within the OTG, in which case it should be the proper new github relative URL (URL Encoded).
- [ ] Foot notes/references should use standard in-line links. (One of the first two here: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links)
    - If a foot note is really required use numeric reference type tags, ex: Content ref: `[[1]]` Footer ref:  `[1]: http://example.org`
- [ ] Images need to be saved to <chapter (or sub-section)>/images/<image.ext> and linked relatively like images/image.ext. (Images in a folder along side the relevant content. [Even if that means duplicates in the overall project.])
- [ ] Any spurious `/` terminating lines, etc. need to be removed and appropriately fixed up for markdown (it's likely a spot that requires a double line-feed prior to an image or other element).
- [ ] In some cases (due to the source of the text) there are broken apostrophes `'` they should just be restored.
- [ ] HTML entities need to be properly handled.
  - Occurrences of `\$` should become `&dollar;`
  - At symbols `@` should become `&commat;`
- [ ] Headers should be properly formatted with leading # (hashtags) per: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers

### PR DESCRIPTION
**Pull request by rejahrehim**
This PR covers issue #89

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Fix linting issues
- Fix dead links

### CODE REVIEW

File: document/5_Reporting/Reporting.md [hunk 1]
```diff
@@ -1,173 +1,165 @@
...
-                   OTG-CLIENT-009                                             Testing for Clickjacking                                                         
-                   OTG-CLIENT-010                                                Testing WebSockets                                                            
-                   OTG-CLIENT-011                                                Test Web Messaging                                                            
-                   OTG-CLIENT-012                                                Test Local Storage                                                            
-  ------------------------------------------------- ---------------------------------------------------------------------------- -------------- -------------- --------------------- --
-
-**Appendix**
+| Test ID  | Test Description | Findings | Severity | Recommendations |
+| ---------| -----------------| -------- | -------- | --------------- |
+| **Information Gathering**  |
```
Comment (ThunderSon) [inline document/5_Reporting/Reporting.md]: @kingthorin what do you think about the following:
```
| **OTG-IFNO** | **Information Gathering** |
```
Might it not be better?

File: document/5_Reporting/Reporting.md [hunk 2]
```diff
@@ -1,173 +1,165 @@
...
--   Screenshots and command lines to indicate what tasks were undertaken during the execution of the test case
--   The affected item
--   A technical description of the issue and the affected function or object
--   A section on resolving the issue
--   The severity rating \[1\], with vector notation if using CVSS
+- Screenshots and command lines to indicate what tasks were undertaken during the execution of the test case
+- The affected item
+- A technical description of the issue and the affected function or object
+- A section on resolving the issue
+- The severity rating, with vector notation if using [CVSS (Industry standard vulnerability severity and risk rankings (CVSS)](http://www.first.org/cvss)
```
Comment (ThunderSon) [inline document/5_Reporting/Reporting.md]: Kindly change from http to https.
You have CVSS at the the start of the link, and at the end. Remove the last one.

Comment (kingthorin) [reply document/5_Reporting/Reporting.md]: Also the round brackets mismatch (open vs closed)

File: document/5_Reporting/Reporting.md (hunk 1 above)
Comment (kingthorin) [reply document/5_Reporting/Reporting.md]: Ya that makes sense

Comment (ThunderSon) [reply document/5_Reporting/Reporting.md]: @rejahrehim can you do that on all the table sections? It looks better visually.

Comment (rejahrehim) [reply document/5_Reporting/Reporting.md]: Sure.

File: document/5_Reporting/Reporting.md (hunk 2 above)
Comment (rejahrehim) [reply document/5_Reporting/Reporting.md]: Done

**Pull request merged**